config/runtime.exs

```diff
@@ -49,7 +50,7 @@ if config_env() == :prod do
   port = String.to_integer(System.get_env("PORT") || "4000")
 
   config :echo, EchoWeb.Endpoint,
-    http: [ip: {127, 0, 0, 1}, port: port],
+    http: [ip: {0, 0, 0, 0}, port: port],
```

Comment:
this needs to be `0.0.0.0` to be reached from the host when running inside the docker container